charts/cortex/values.yaml

```diff
@@ -1,3 +1,11 @@
+topologyAware:
+    enabled: true
+    availabilityCells:
+    - netic-dc1
```

Comment:
I do not think we want to put default Netic values into a public chart here. Possibly I think the default should be that topologyAware is not enabled as this is a pretty advanced use case.